src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -397,6 +350,23 @@ typedef struct {
     char kvsname[MPIDI_KVSAPPSTRLEN];
     char pname[MPI_MAX_PROCESSOR_NAME];
     int port_name_tag_mask[MPIR_MAX_CONTEXT_MASK];
+
+    /* Capability settings */
+#ifdef MPIDI_OFI_ENABLE_RUNTIME_CHECKS
```

Comment:
Turns out my preprocessor-fu isn't as good as I hoped. This is more or less what I'm trying to do. Hopefully I get some pointers: http://stackoverflow.com/questions/40451392/use-multiple-macros-in-a-macro-definition. You can probably extrapolate to figure out where I'm going with this.